tests/packagedcode/data/alpine/apkbuild-problems/alpine14/main/sudo/APKBUILD-expected.json

```diff
@@ -86,5 +86,6 @@
   "repository_download_url": null,
   "api_data_url": null,
   "datasource_id": "alpine_apkbuild",
+  "holder": null,
```

Comment:
The `holder` attribute has not gone up with `copyright` as these tests don't fail for attribute order? We should ideally force regen anything with package scan on as otherwise we would have these changes pop up later when we regen for other cases and this would be more churn IMHO

Reply:
Done